sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -491,7 +496,7 @@ def get_rx_power(self):
         return rx_power
 
     def get_rx_power_support(self):
-        return not self.is_flat_memory()
+        return not self.is_flat_memory() and self.xcvr_eeprom.read(consts.RX_POWER_SUPPORT_FIELD)
```

Comment:
this field is inside reg group MODULE_CHAR_ADVT_FIELD so cannot be read directly.